eng/scripts/Create-Apiview-Token-Files.ps1

```diff
@@ -31,20 +32,30 @@ if ($reviews -ne $null)
         azcopy cp "$sourcePath" $codeDir/$($r.FileName) --recursive=true
 
         #Create staging path for review and revision ID
-        $CodeFilePath = Join-Path -Path $StagingPath $r.ReviewID | Join-Path -ChildPath $r.RevisionID
-        if (-not (Test-Path -Path $CodeFilePath)) {
-            New-Item -Path $CodeFilePath -ItemType Directory
+        $CodeFileDirectory = Join-Path -Path $StagingPath $r.ReviewID | Join-Path -ChildPath $r.RevisionID
+        if (-not (Test-Path -Path $CodeFileDirectory)) {
+            New-Item -Path $CodeFileDirectory -ItemType Directory
         }
 
         $reviewGenScriptPath = Join-Path $PSScriptRoot $ApiviewGenScript
         if ($ParserPath -eq "")
         {
-            &($reviewGenScriptPath) -SourcePath $codeDir/$($r.FileName) -OutPath $CodeFilePath
+            &($reviewGenScriptPath) -SourcePath $codeDir/$($r.FileName) -OutPath $CodeFileDirectory
         }
         else
         {
-            &($reviewGenScriptPath) -SourcePath $codeDir/$($r.FileName) -OutPath $CodeFilePath -ParserPath $ParserPath
+            &($reviewGenScriptPath) -SourcePath $codeDir/$($r.FileName) -OutPath $CodeFileDirectory -ParserPath $ParserPath
         }
+
+        if ((Get-ChildItem -Path $CodeFileDirectory -Recurse | Measure-Object).Count -gt 0)
+        {
+            $wasApiviewTokenFileCreated = $true
+        }
+    }
+    if (-not $wasApiviewTokenFileCreated)
+    {
+        Write-Host "No Apiview Token files were created"
+        exit 1;
```

Comment:
This will be an issue when it processes more than one review. It will avoid processing further reviews in the request. This same script is used for batch update when Python API reviews are upgraded.